extensions/amp-ad-network-adsense-impl/0.1/adsense-a4a-config.js

```diff
@@ -64,8 +65,11 @@ export const URL_EXPERIMENT_MAPPING = {
  * @returns {boolean}
  */
 export function adsenseIsA4AEnabled(win, element) {
+  // TODO(@taymonbeal, #10524): unify this with methods in AmpA4A
```

Comment:
I suggest as part of the refactor we make getFromJsonThenAttribute(json, element, dataType) which you could reference here and within A4A